backend/capellacollab/sessions/operators/helper.py

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import inspect
+import typing as t
+
+
+def get_source_of_python_function(function: t.Callable):
```

Comment:
I'd move this "helper" into k8s.py.

"helper" is a bad name for any module.

Reply:
The `k8s.py` has 1030 lines already and is too large imho. Also, it's not related to the k8s operator itself - it just returns the source of a python function.